thirdeye/thirdeye-frontend/app/pods/manage/alert/template.hbs

```diff
@@ -1,75 +1,79 @@
-<section class="te-search-results te-alert-page-top">
+<section class="te-alert-page__top te-search-results">
   <div class="container">
     <div class="te-search-results__header">
       <div class="te-search-results__title-group">
         <div class="te-search-results__title">
           <span title={{alertData.functionName}}>
             {{alertData.functionName}}
           </span>
-          <span class="te-search-results__tag {{if alertData.isActive "te-search-results__tag--active"}}">
+          <div class="te-search-results__tag {{if alertData.isActive "te-search-results__tag--active"}}">
             {{if alertData.isActive "Active" "Inactive"}}
-          </span>
+          </div>
         </div>
       </div>
+        <div class="te-search-results__cta">
+          {{#link-to "manage.alerts.edit" alertData.id tagName="button" type="button" class="te-button te-button--sub"}}Edit{{/link-to}}
+          <button {{action "onClickAlertSubscribe"}} class="te-button te-button--main">Subscribe</button>
+        </div>
     </div>
     <ul class="te-search-results__list te-search-results__list--details-block row">
       <div class="col-xs-12 col-sm-5">
         <li class="te-search-results__row">
           {{!-- Todo: Dry this out --}}
-          <div class="te-search-results__option">Metric</div>
-          <div class="te-search-results__value" title={{alertData.metric}}>{{alertData.metric}}</div>
+          <div class="te-search-results__option-solo">Metric</div>
```

Comment:
is -solo a modifier or a new block element? 
if it's the former, let's use the correct BEM syntax : `te-search-results__option--solo`

Then the usage will be as follows: (base class + class modifier)
```
class="te-search-results__option te-search-results__option--solo"
```

Reply:
I want it to be a modifier - I've changed the block element and downstream template.